.readme-partials.yaml

```diff
@@ -110,28 +110,33 @@ custom_content: |
   -------
   In this feature launch, the [Java Datastore client](https://github.com/googleapis/java-datastore) now offers gRPC as a transport layer option with experimental support. Using [gRPC connection pooling](https://grpc.io/docs/guides/performance/) enables distributing RPCs over multiple connections which may improve performance.
   
-  #### Download Instructions
-  Instructions:
-  1. Clone the grpc-experimental branch from GitHub:
-  ```python
-  git clone -b grpc-experimental https://github.com/googleapis/java-datastore.git
-  ```
-  2. Run the following commands to build the library:
-  ```python
-  # Go to the directory the code was downloaded to
-  cd java-datastore/
-  
-  # Build the library
-  mvn clean install -DskipTests=true
-  ```
-  3. Add the following dependency to your project:
-  ```xml
-  <dependency>
+  #### Installation Instructions
+  The client can be built from the `grpc-experimental` branch on GitHub. For private preview, you can also download the artifact with the instructions provided below.
+
+  1. Download the datastore private preview package with dependencies:
+    ```python
```

Comment:
Why python? Nintendo are Java users. If their client vms don't have python - can we use standard unix tools like `wget -P` or `curl -o`